src/main/config/logback.xml

```diff
@@ -20,27 +20,31 @@
     </encoder>
   </appender>
 
-
-  <appender name="OBJECT-METRICS" class="ch.qos.logback.core.ConsoleAppender">
-    <!--<appender name="OBJECT-METRICS" class="ch.qos.logback.core.rolling.RollingFileAppender">-->
-    <encoder class="net.logstash.logback.encoder.LogstashEncoder">
-      <customFields>{"node":"${emissary.node.name}","port":"${emissary.node.port}","application":"server"}</customFields>
-      <includeMdc>false</includeMdc>
-      <fieldNames>
-        <version>[ignore]</version>
-        <logValue>[ignore]</logValue>
-        <logger>[ignore]</logger>
-        <thread>[ignore]</thread>
-        <levelValue>[ignore]</levelValue>
-        <level>[ignore]</level>
-        <message>[ignore]</message>
-      </fieldNames>
-    </encoder>
-    <!--<file>logs/${emissary.node.name}-${emissary.node.port}-object-metrics.log</file>
-    <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
-      <fileNamePattern>logs/${emissary.node.name}-${emissary.node.port}-object-metrics.log.%d{yyyy-MM-dd-HH}</fileNamePattern>
-    </rollingPolicy>-->
-  </appender>
+  <!-- Enable output metrics logging by creating an environment variable of "LogObjectMetrics=true" -->
+  <if condition='${LogObjectMetrics} == true'>
```

Comment:
Should we use the env variable in the config to turn on/off the metrics as well?? Something like -
```
OUTPUT_OBJECT_METRICS = "@ENV{'LogObjectMetrics'}"
```

Reply:
Forgot that env's can be in configs! I like this idea. No need for custom flavors as well. I will make the change and validate